src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditorUtils.cs

```diff
@@ -68,7 +68,7 @@ public static HighlightingRule CreateClassHighlightRule(EngineController engineC
             };
 
             var wordList = engineController.CodeCompletionServices.GetClasses();
-            String regex = String.Format(@"\b({0})({0})?\b", String.Join("|", wordList));
```

Comment:
This pattern was incorrect earlier. It matched things like `PointPoint` and `PointLine`, which was the wrong behaviour.